jest.config.js

```diff
@@ -5,6 +5,7 @@
 const JEST_PROJECT = process.env.JEST_PROJECT
 
 module.exports = {
+  notify: true,
```

Comment:
Does it display desktop notifications when tests are failing?

Reply:
No, I tried to test but did not have notifications actually, I am removing this part. I'll try again later.